content/en/security_platform/application_security/troubleshooting.md

```diff
@@ -0,0 +1,507 @@
+---
+
+
+## Overview
+
+If you experience unexpected behavior with Datadog Application Security, there are common issues you can investigate, as mentioned below. If you continue to have trouble, reach out to [Datadog support][1] for further assistance. 
+
+## Application Security rate limits
+
+Application Security traces are rate-limited to 100 traces per second. Traces sent after the limit are not reported. Contact [Datadog support][1] if you need to change the limit.
+
+## No Suspicious Requests detected by Application Security
+
+There are a series of steps that must run successfully for threat information to appear in the Application Security [Trace and Signals Explorer][2]. It is important to check each step when investigating this issue. Additional troubleshooting steps for specific languages are in the language tab at the end.
+
+### Confirm Application Security is enabled
+
+You can use the metric `datadog.apm.appsec_host` to check if Application Security is running.
+
+1. Go to **Metrics > Summary** in Datadog.
+2. Search for the metric `datadog.apm.appsec_host`. If the metric doesn’t exist, then there are no services running Application Security. If the metric exists, the services are reported with the metric tags `host` and `service`.
+3. Select the metric, and in the **Tags** section, search for `service` to see which services are running Application Security.
+
+If you are not seeing `datadog.apm.appsec_host`, check the [in-app instructions][3] to confirm that all steps for the initial setup are complete.
+
+Application Security data is sent with APM traces. See [APM troubleshooting][4] to [confirm APM setup][5] and check for [connection errors][6].
+
+### Send a test attack to your application
+
+ To test your Application Security setup, trigger the [Security Scanner Detected][7] rule by running a file that contains the following curl script: 
+
+{{< programming-lang-wrapper langs="java,.NET,go,ruby,PHP,NodeJS" >}}
+{{< programming-lang lang="java" >}}
+
+```bash
+for ((i=1;i<=200;i++)); 
+do
+# Target existing service’s routes
+curl https://your-application-url/existing-route -A dd-test-scanner-log;
+# Target non existing service’s routes
+curl https://your-application-url/non-existing-route -A dd-test-scanner-log;
+done
+```
+
+**Note:** The `dd-test-scanner-log` value is supported in the most recent releases and in the application security event configuration file version >= 1.2.5.
+
+{{< /programming-lang >}}
+{{< programming-lang lang=".NET" >}}
+
+```bash
+for ((i=1;i<=200;i++)); 
+do
+# Target existing service’s routes
+curl https://your-application-url/existing-route -A dd-test-scanner-log;
+# Target non existing service’s routes
+curl https://your-application-url/non-existing-route -A dd-test-scanner-log;
+done
+```
+
+**Note:** The `dd-test-scanner-log` value is supported in the most recent releases and in the application security event configuration file version >= 1.2.5.
```

Comment:
Is this version visible anywhere by the customer? It might be confusing to mention a version they can't check.

Reply:
Hey @lloeki let me know if this should be removed. It's also mentioned in the Setup and Configuration page. https://docs.datadoghq.com/security_platform/application_security/getting_started/java?tab=dockercli